server.js

```diff
@@ -196,6 +196,9 @@ const chooseAllForm = pug.compileFile('views/question_types/chooseAll-answer.pug
 const matchingForm = pug.compileFile('views/question_types/matching-answer.pug');
 const orderingForm = pug.compileFile('views/question_types/ordering-answer.pug');
 const leaderboardTable = pug.compileFile('views/leaderboard-table.pug');
+const pointsTable = pug.compileFile('views/points-leaderboard.pug');
```

Comment:
move all of these inside a folder called leaderboard (you will have to make this folder)

Reply:
all of these === any pug file related to leaderboard